contracts/english_auction_rev1.1.scilla

```diff
@@ -143,11 +159,17 @@ let make_error =
 
 contract EnglishAuctionOrderbook
 (
-  initial_contract_owner: ByStr20
+  initial_contract_owner: ByStr20,
+  collection_contract: ByStr20 with contract
```

Comment:
@sam-zilliqa it would be safer to use `initial_collection_contract` name, for readability and to avoid confusion (see my comment below)

Reply:
Updated